uniffi_bindgen/src/scaffolding/templates/ObjectTemplate.rs

```diff
@@ -55,7 +55,7 @@ pub extern "C" fn {{ ffi_free.name() }}(ptr: *const std::os::raw::c_void, call_s
         uniffi::rust_call(call_status, || {
             {{ cons|return_ffi_converter }}::lower_return(
                 {%- if cons.throws() %}
-                {{ obj.rust_name() }}::{% call rs::to_rs_call(cons) %}.map(::std::sync::Arc::new).map_err(Into::into)
+                {{ obj.rust_name() }}::{% call rs::to_rs_call(cons) %}.map(::std::sync::Arc::new)
```

Comment:
I feel good about this change, but maybe it breaks backwards comparability?  If so, it should have a changelog entry.